### ISSUE TITLE
Database error

### ISSUE THREAD

**Issue by InnerFireZ** [LuatixHQ/opencti#5660]
## Prerequisites

- [ X] I read the [Deployment and Setup](https://docs.opencti.io/latest/deployment/overview) section of the OpenCTI documentation as well as the [Troubleshooting](https://docs.opencti.io/latest/deployment/troubleshooting) page and didn't find anything relevant to my problem.
- [ X] I went through old GitHub issues and couldn't find anything relevant
- [X ] I googled the issue and didn't find anything relevant

## Description

Hello my instance of opencti break itself and now logs in opencti pladform caontainer look like this:

`{"category":"APP","errors":[{"attributes":{"genre":"TECHNICAL","http_status":500},"message":"Engine attachment processor configuration fail","name":"CONFIGURATION_ERROR","stack":"CONFIGURATION_ERROR: Engine attachment processor configuration fail\n    at error (/opt/opencti/build/src/config/errors.js:8:10)\n    at ConfigurationError (/opt/opencti/build/src/config/errors.js:64:53)\n    at /opt/opencti/build/src/database/engine.js:787:20\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at elConfigureAttachmentProcessor (/opt/opencti/build/src/database/engine.js:775:5)\n    at searchEngineInit (/opt/opencti/build/src/database/engine.js:264:32)\n    at checkSystemDependencies (/opt/opencti/build/src/initialization.js:127:3)\n    at platformStart (/opt/opencti/build/src/boot.js:234:5)"},{"message":"parse_exception\n\tRoot causes:\n\t\tparse_exception: No processor type exists with name [attachment]","name":"ResponseError","stack":"ResponseError: parse_exception\n\tRoot causes:\n\t\tparse_exception: No processor type exists with name [attachment]\n    at kJt.request (/opt/opencti/build/node_modules/@elastic/transport/src/Transport.ts:553:17)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at PZt.putPipeline (/opt/opencti/build/node_modules/@elastic/elasticsearch/src/api/api/ingest.ts:195:12)\n    at elConfigureAttachmentProcessor (/opt/opencti/build/src/database/engine.js:775:5)\n    at searchEngineInit (/opt/opencti/build/src/database/engine.js:264:32)\n    at checkSystemDependencies (/opt/opencti/build/src/initialization.js:127:3)\n    at platformStart (/opt/opencti/build/src/boot.js:234:5)"}],"level":"error","message":"Engine attachment processor configuration fail","timestamp":"2024-01-23T13:57:21.452Z","version":"5.12.21"}
{"category":"APP","errors":[{"attributes":{"genre":"TECHNICAL","http_status":500},"message":"Engine attachment processor configuration fail","name":"CONFIGURATION_ERROR","stack":"CONFIGURATION_ERROR: Engine attachment processor configuration fail\n    at error (/opt/opencti/build/src/config/errors.js:8:10)\n    at ConfigurationError (/opt/opencti/build/src/config/errors.js:64:53)\n    at /opt/opencti/build/src/database/engine.js:787:20\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at elConfigureAttachmentProcessor (/opt/opencti/build/src/database/engine.js:775:5)\n    at searchEngineInit (/opt/opencti/build/src/database/engine.js:264:32)\n    at checkSystemDependencies (/opt/opencti/build/src/initialization.js:127:3)\n    at platformStart (/opt/opencti/build/src/boot.js:234:5)"},{"message":"parse_exception\n\tRoot causes:\n\t\tparse_exception: No processor type exists with name [attachment]","name":"ResponseError","stack":"ResponseError: parse_exception\n\tRoot causes:\n\t\tparse_exception: No processor type exists with name [attachment]\n    at kJt.request (/opt/opencti/build/node_modules/@elastic/transport/src/Transport.ts:553:17)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at PZt.putPipeline (/opt/opencti/build/node_modules/@elastic/elasticsearch/src/api/api/ingest.ts:195:12)\n    at elConfigureAttachmentProcessor (/opt/opencti/build/src/database/engine.js:775:5)\n    at searchEngineInit (/opt/opencti/build/src/database/engine.js:264:32)\n    at checkSystemDependencies (/opt/opencti/build/src/initialization.js:127:3)\n    at platformStart (/opt/opencti/build/src/boot.js:234:5)"}],"level":"error","message":"Engine attachment processor configuration fail","timestamp":"2024-01-23T13:58:45.015Z","version":"5.12.21"}
ory":"APP","errors":[{"attributes":{"body":{"script":{"params":{"entity_type":"User","internal_id":"88ec0c6a-13ce-5e39-b486-354fe4a7084f","password":"#########","updated_at":"2024-01-23T13:58:34.633Z"},"source":"ctx._source['internal_id'] = params['internal_id'];ctx._source['entity_type'] = params['entity_type'];ctx._source['password'] = params['password'];ctx._source['updated_at'] = params['updated_at']"}},"documentId":"88ec0c6a-13ce-5e39-b486-354fe4a7084f","genre":"TECHNICAL","http_status":500},"message":"Update indexing fail","name":"DATABASE_ERROR","stack":"DATABASE_ERROR: Update indexing fail\n    at error (/opt/opencti/build/src/config/errors.js:8:10)\n    at DatabaseError (/opt/opencti/build/src/config/errors.js:58:48)\n    at /opt/opencti/build/src/database/engine.js:2631:11\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async Promise.all (index 0)\n    at updateAttributeMetaResolved (/opt/opencti/build/src/database/middleware.js:1942:7)\n    at initAdmin (/opt/opencti/build/src/domain/user.js:1386:5)\n    at initializeAdminUser (/opt/opencti/build/src/config/providers.js:45:5)\n    at platformInit (/opt/opencti/build/src/initialization.js:379:7)\n    at platformStart (/opt/opencti/build/src/boot.js:238:5)"},{"message":"cluster_block_exception\n\tRoot causes:\n\t\tcluster_block_exception: index [opencti_internal_objects-000001] blocked by: [TOO_MANY_REQUESTS/12/disk usage exceeded flood-stage watermark, index has read-only-allow-delete block];","name":"ResponseError","stack":"ResponseError: cluster_block_exception\n\tRoot causes:\n\t\tcluster_block_exception: index [opencti_internal_objects-000001] blocked by: [TOO_MANY_REQUESTS/12/disk usage exceeded flood-stage watermark, index has read-only-allow-delete block];\n    at kJt.request (/opt/opencti/build/node_modules/@elastic/transport/src/Transport.ts:553:17)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at W$e.UpdateApi [as update] (/opt/opencti/build/node_modules/@elastic/elasticsearch/src/api/api/update.ts:75:10)\n    at async Promise.all (index 0)\n    at updateAttributeMetaResolved (/opt/opencti/build/src/database/middleware.js:1942:7)\n    at initAdmin (/opt/opencti/build/src/domain/user.js:1386:5)\n    at initializeAdminUser (/opt/opencti/build/src/config/providers.js:45:5)\n    at platformInit (/opt/opencti/build/src/initialization.js:379:7)\n    at platformStart (/opt/opencti/build/src/boot.js:238:5)"}],"level":"error","message":"Update indexing fail","timestamp":"2024-01-23T13:58:34.648Z","version":"5.12.21"}
{"category":"APP","errors":[{"attributes":{"genre":"TECHNICAL","http_status":500},"message":"Engine attachment processor configuration fail","name":"CONFIGURATION_ERROR","stack":"CONFIGURATION_ERROR: Engine attachment processor configuration fail\n    at error (/opt/opencti/build/src/config/errors.js:8:10)\n    at ConfigurationError (/opt/opencti/build/src/config/errors.js:64:53)\n    at /opt/opencti/build/src/database/engine.js:787:20\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at elConfigureAttachmentProcessor (/opt/opencti/build/src/database/engine.js:775:5)\n    at searchEngineInit (/opt/opencti/build/src/database/engine.js:264:32)\n    at checkSystemDependencies (/opt/opencti/build/src/initialization.js:127:3)\n    at platformStart (/opt/opencti/build/src/boot.js:234:5)"},{"message":"parse_exception\n\tRoot causes:\n\t\tparse_exception: No processor type exists with name [attachment]","name":"ResponseError","stack":"ResponseError: parse_exception\n\tRoot causes:\n\t\tparse_exception: No processor type exists with name [attachment]\n    at kJt.request (/opt/opencti/build/node_modules/@elastic/transport/src/Transport.ts:553:17)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at PZt.putPipeline (/opt/opencti/build/node_modules/@elastic/elasticsearch/src/api/api/ingest.ts:195:12)\n    at elConfigureAttachmentProcessor (/opt/opencti/build/src/database/engine.js:775:5)\n    at searchEngineInit (/opt/opencti/build/src/database/engine.js:264:32)\n    at checkSystemDependencies (/opt/opencti/build/src/initialization.js:127:3)\n    at platformStart (/opt/opencti/build/src/boot.js:234:5)"}],"level":"error","message":"Engine attachment processor configuration fail","timestamp":"2024-01-23T13:58:58.966Z","version":"5.12.21"}
ory":"APP","errors":[{"attributes":{"body":{"script":{"params":{"entity_type":"User","internal_id":"88ec0c6a-13ce-5e39-b486-354fe4a7084f","password":"#########","updated_at":"2024-01-23T13:58:48.217Z"},"source":"ctx._source['internal_id'] = params['internal_id'];ctx._source['entity_type'] = params['entity_type'];ctx._source['password'] = params['password'];ctx._source['updated_at'] = params['updated_at']"}},"documentId":"88ec0c6a-13ce-5e39-b486-354fe4a7084f","genre":"TECHNICAL","http_status":500},"message":"Update indexing fail","name":"DATABASE_ERROR","stack":"DATABASE_ERROR: Update indexing fail\n    at error (/opt/opencti/build/src/config/errors.js:8:10)\n    at DatabaseError (/opt/opencti/build/src/config/errors.js:58:48)\n    at /opt/opencti/build/src/database/engine.js:2631:11\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async Promise.all (index 0)\n    at updateAttributeMetaResolved (/opt/opencti/build/src/database/middleware.js:1942:7)\n    at initAdmin (/opt/opencti/build/src/domain/user.js:1386:5)\n    at initializeAdminUser (/opt/opencti/build/src/config/providers.js:45:5)\n    at platformInit (/opt/opencti/build/src/initialization.js:379:7)\n    at platformStart (/opt/opencti/build/src/boot.js:238:5)"},{"message":"cluster_block_exception\n\tRoot causes:\n\t\tcluster_block_exception: index [opencti_internal_objects-000001] blocked by: [TOO_MANY_REQUESTS/12/disk usage exceeded flood-stage watermark, index has read-only-allow-delete block];","name":"ResponseError","stack":"ResponseError: cluster_block_exception\n\tRoot causes:\n\t\tcluster_block_exception: index [opencti_internal_objects-000001] blocked by: [TOO_MANY_REQUESTS/12/disk usage exceeded flood-stage watermark, index has read-only-allow-delete block];\n    at kJt.request (/opt/opencti/build/node_modules/@elastic/transport/src/Transport.ts:553:17)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at W$e.UpdateApi [as update] (/opt/opencti/build/node_modules/@elastic/elasticsearch/src/api/api/update.ts:75:10)\n    at async Promise.all (index 0)\n    at updateAttributeMetaResolved (/opt/opencti/build/src/database/middleware.js:1942:7)\n    at initAdmin (/opt/opencti/build/src/domain/user.js:1386:5)\n    at initializeAdminUser (/opt/opencti/build/src/config/providers.js:45:5)\n    at platformInit (/opt/opencti/build/src/initialization.js:379:7)\n    at platformStart (/opt/opencti/build/src/boot.js:238:5)"}],"level":"error","message":"Update indexing fail","timestamp":"2024-01-23T13:58:48.233Z","version":"5.12.21"}`

`ory":"APP","errors":[{"attributes":{"body":{"script":{"params":{"entity_type":"User","internal_id":"88ec0c6a-13ce-5e39-b486-354fe4a7084f","password":"#####.","updated_at":"2024-01-23T14:05:33.398Z"},"source":"ctx._source['internal_id'] = params['internal_id'];ctx._source['entity_type'] = params['entity_type'];ctx._source['password'] = params['password'];ctx._source['updated_at'] = params['updated_at']"}},"documentId":"88ec0c6a-13ce-5e39-b486-354fe4a7084f","genre":"TECHNICAL","http_status":500},"message":"Update indexing fail","name":"DATABASE_ERROR","stack":"DATABASE_ERROR: Update indexing fail\n    at error (/opt/opencti/build/src/config/errors.js:8:10)\n    at DatabaseError (/opt/opencti/build/src/config/errors.js:58:48)\n    at /opt/opencti/build/src/database/engine.js:2631:11\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async Promise.all (index 0)\n    at updateAttributeMetaResolved (/opt/opencti/build/src/database/middleware.js:1942:7)\n    at initAdmin (/opt/opencti/build/src/domain/user.js:1386:5)\n    at initializeAdminUser (/opt/opencti/build/src/config/providers.js:45:5)\n    at platformInit (/opt/opencti/build/src/initialization.js:379:7)\n    at platformStart (/opt/opencti/build/src/boot.js:238:5)"},{"message":"cluster_block_exception\n\tRoot causes:\n\t\tcluster_block_exception: index [opencti_internal_objects-000001] blocked by: [TOO_MANY_REQUESTS/12/disk usage exceeded flood-stage watermark, index has read-only-allow-delete block];","name":"ResponseError","stack":"ResponseError: cluster_block_exception\n\tRoot causes:\n\t\tcluster_block_exception: index [opencti_internal_objects-000001] blocked by: [TOO_MANY_REQUESTS/12/disk usage exceeded flood-stage watermark, index has read-only-allow-delete block];\n    at kJt.request (/opt/opencti/build/node_modules/@elastic/transport/src/Transport.ts:553:17)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at W$e.UpdateApi [as update] (/opt/opencti/build/node_modules/@elastic/elasticsearch/src/api/api/update.ts:75:10)\n    at async Promise.all (index 0)\n    at updateAttributeMetaResolved (/opt/opencti/build/src/database/middleware.js:1942:7)\n    at initAdmin (/opt/opencti/build/src/domain/user.js:1386:5)\n    at initializeAdminUser (/opt/opencti/build/src/config/providers.js:45:5)\n    at platformInit (/opt/opencti/build/src/initialization.js:379:7)\n    at platformStart (/opt/opencti/build/src/boot.js:238:5)"}],"level":"error","message":"Update indexing fail","timestamp":"2024-01-23T14:05:33.416Z","version":"5.12.21"}`

Can someone explain me how to fix this?
## Environment

1. OS (where OpenCTI server runs): Linux opencti 5.13.0-52-generic
2. OpenCTI version: 5.12.21
3. OpenCTI client: frontend
4. Other environment details:
Im using portainer to manage all of the containers

## Additional information

Now webservice wich is on port 8080 wont start...

**Comment by nino-filigran** [comment]
@innerfirebg if you don't find the needed support, feel also free to drop a note in the Troubleshooting slack channel.

**Comment by aptohunter** [comment]
Have you solved the problem yet

**Comment by menmvlad** [comment]
Hello, we have installed opencti with portainer and bumped into the same issue with the following error: 

`{"category":"APP","errors":[{"attributes":{"genre":"TECHNICAL","http_status":500},"message":"Search engine seems down","name":"CONFIGURATION_ERROR","stack":"CONFIGURATION_ERROR: Search engine seems down\n    at error (/opt/opencti/build/src/config/errors.js:8:10)\n    at ConfigurationError (/opt/opencti/build/src/config/errors.js:64:53)\n    at /opt/opencti/build/src/database/engine.js:180:15\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at searchEngineVersion (/opt/opencti/build/src/database/engine.js:176:22)\n    at searchEngineInit (/opt/opencti/build/src/database/engine.js:219:27)\n    at checkSystemDependencies (/opt/opencti/build/src/initialization.js:127:3)\n    at platformStart (/opt/opencti/build/src/boot.js:228:5)"},{"message":"Request timed out","name":"TimeoutError","stack":"TimeoutError: Request timed out\n    at ClientRequest.onTimeout (/opt/opencti/build/node_modules/@opensearch-project/opensearch/lib/Connection.js:123:16)\n    at ClientRequest.emit (node:events:514:28)\n    at Socket.emitRequestTimeout (node:_http_client:840:9)\n    at Object.onceWrapper (node:events:628:28)\n    at Socket.emit (node:events:526:35)\n    at Socket._onTimeout (node:net:589:8)\n    at listOnTimeout (node:internal/timers:573:17)\n    at processTimers (node:internal/timers:514:7)"}],"level":"error","message":"Search engine seems down","timestamp":"2024-02-08T12:39:36.151Z","version":"5.12.15"}`

I'm unable to locallize the engine.js file and the others as I cannot find them on the linux machines and I think the problem may be because elasticsearch is located in the second host, the worker and the platform is located on the manager. And also we want to distribute the containers manually because of the resource. Do you know if its possible? Our manager is a VM with 4 CPUs, 12 RAM and 120GB HDD and its CPU and RAM are almost permanently overloaded, that's why we decided to create a second host.

Thanks.

**Comment by V1D1AN** [comment]
Hi, I have the same problem since the version 6.2.12 ... My OpenCTI don't start.
But i changed the level of log and i see the opencti work for starting ... and i see in my kibana that my index "opencti_history-000001" change.
I think, i must wait for opencti finished the modification in database for starting



```
{"category":"APP","level":"info","message":"[SEARCH] Engine detected to elk","source":"backend","timestamp":"2024-08-20T07:35:59.466Z","version":"6.2.14"}
{"category":"APP","errors":[{"attributes":{"genre":"TECHNICAL","http_status":500},"message":"Engine attachment processor configuration fail","name":"CONFIGURATION_ERROR","stack":"GraphQLError: Engine attachment processor configuration fail\n    at error (/opt/opencti/build/src/config/errors.js:7:10)\n    at ConfigurationError (/opt/opencti/build/src/config/errors.js:75:53)\n    at /opt/opencti/build/src/database/engine.js:879:20\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at elConfigureAttachmentProcessor (/opt/opencti/build/src/database/engine.js:867:5)\n    at searchEngineInit (/opt/opencti/build/src/database/engine.js:332:32)\n    at checkSystemDependencies (/opt/opencti/build/src/initialization.js:38:3)\n    at platformStart (/opt/opencti/build/src/boot.js:15:7)"},{"message":"parse_exception\n\tRoot causes:\n\t\tparse_exception: No processor type exists with name [attachment]","name":"ResponseError","stack":"ResponseError: parse_exception\n\tRoot causes:\n\t\tparse_exception: No processor type exists with name [attachment]\n    at zzt._request (/opt/opencti/build/node_modules/@elastic/transport/src/Transport.ts:601:17)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at /opt/opencti/build/node_modules/@elastic/transport/src/Transport.ts:704:22\n    at zzt.request (/opt/opencti/build/node_modules/@elastic/transport/src/Transport.ts:701:14)\n    at g$t.putPipeline (/opt/opencti/build/node_modules/@elastic/elasticsearch/src/api/api/ingest.ts:323:12)\n    at elConfigureAttachmentProcessor (/opt/opencti/build/src/database/engine.js:867:5)\n    at searchEngineInit (/opt/opencti/build/src/database/engine.js:332:32)\n    at checkSystemDependencies (/opt/opencti/build/src/initialization.js:38:3)\n    at platformStart (/opt/opencti/build/src/boot.js:15:7)"}],"level":"error","message":"Engine attachment processor configuration fail","source":"backend","timestamp":"2024-08-20T07:35:59.529Z","version":"6.2.14"}
...
...
...
{"category":"APP","level":"info","message":"[OPENCTI-MODULE] INGESTION - Running ingestion handlers","source":"backend","timestamp":"2024-08-20T07:41:22.065Z","version":"6.2.14"}
{"category":"APP","level":"info","message":"[OPENCTI] Expiration manager will revoke 12 elements","source":"backend","timestamp":"2024-08-20T07:41:22.118Z","version":"6.2.14"}
{"category":"APP","level":"info","message":"[OPENCTI-MODULE] INGESTION - Running ingestion handlers","source":"backend","timestamp":"2024-08-20T07:46:22.065Z","version":"6.2.14"}
{"category":"APP","level":"info","message":"[OPENCTI] Expiration manager will revoke 4 elements","source":"backend","timestamp":"2024-08-20T07:46:23.019Z","version":"6.2.14"}
{"category":"APP","level":"info","message":"[OPENCTI-MODULE] INGESTION - Running ingestion handlers","source":"backend","timestamp":"2024-08-20T07:51:22.065Z","version":"6.2.14"}
{"category":"APP","level":"info","message":"[OPENCTI-MODULE] INGESTION - Running ingestion handlers","source":"backend","timestamp":"2024-08-20T07:56:22.065Z","version":"6.2.14"}
```

**Comment by aptohunter** [comment]
你好，你的收信我已收到，谢谢

**Comment by kurtrwall** [comment]
On 6.2.13, I'm also having this problem, I even have the same error message as @V1D1AN 

The offending request at [database/engine.js#867](https://github.com/OpenCTI-Platform/opencti/blob/6.2.13/opencti-platform/opencti-graphql/src/database/engine.js#L867) which I've traced to the [ES client library putPipeline fn](https://github.com/elastic/elasticsearch-js/blob/715292b5013c3d215879ea917e17d4730d78064a/src/api/api/ingest.ts#L323).

In short, it looks like OpenCTI is encountering an issue creating an Ingest Pipeline in my ES cluster. There isn't any auth required or anything, but since OpenCTI is using an ElkClient, I suspect that it needs one? I'm really not sure what to do beyond tweaking configuration around or tracing the name of the Ingest Pipeline OpenCTI is trying to create and creating it manually? 

Help is appreciated!

**Comment by kurtrwall** [comment]
I've just found what the issue is. 

My ES cluster is on version 7.17.x and the required processor "attachment" is unavailable until version 8. In the version requirements matrix, you guys mention Elasticsearch >=8.0, so this is just an oversight for me overall. 

Hope this helps any of you still troubleshooting.

**Comment by V1D1AN** [comment]
Hello @kurtrwall


I'm sad because I made the same analysis on my side. I suspected it because my elastic nodes are also in 7 and when I saw the compatibility matrix, I saw that version 7.x had disappeared.

I use “watchtower” to automatically update my OpenCTI with the latest versions. I had to lose all my OpenCTI data to go back to a version 6.2.11 still compatible with Elasticsearch 7.
As soon as I tried to migrate to 6.2.12, my firefox wouldn't change the page, and I got a 404 for GraphQL Javascript.


So I'm starting from scratch, I'm disappointed and now I've fixed my version of OpenCTI....

**Comment by menmvlad** [comment]
Hi! Can I ask you to stop sending mails to me?

Thanks in advance! Sorry!


Regards,

***@***.***

Kristian Kiradzhiyski
Information Security Analyst @ Mnemonica JSC

Address: 12 Nikolay Haytov str., floor 4, office 13, 1113 Sofia, Bulgaria

***@***.***<https://www.mnemonica.bg/>
***@***.***<https://www.facebook.com/MnemonicaJSC>
***@***.***<https://www.linkedin.com/company/mnemonica-jsc/>

**Comment by V1D1AN** [comment]
@menmvlad You receive mail because you have post a message on https://github.com/OpenCTI-Platform/opencti/issues/5660
You must go on github and stop that..

**Comment by kurtrwall** [comment]
@V1D1AN sounds rough. How did you figure that downgrading to OpenCTI v6.2.11 is compatible with Elasticsearch? I can't find anything supporting that, and the compatibility matrix in the documentation for that version pins at at >=8.0

**Comment by kurtrwall** [comment]
I looked a bit harder, found a version of the platform's dependencies where it pins the elasticsearch client. The last time elasticsearch v7 shows up is at OpenCTI v5.3.7

https://github.com/OpenCTI-Platform/opencti/blob/5.3.7/opencti-platform/opencti-graphql/package.json

**Comment by V1D1AN** [comment]
@kurtrwall , sorry for the quality ...

![image](https://github.com/user-attachments/assets/7581dafc-b7e6-4953-bcbe-23525bd282ed)

**Comment by alice-debra** [comment]
Hello,
We’re following up to see if you still need an answer to this question. If you still need help with this, don't hesitate to ask on our Slack: https://filigran-community.slack.com/. 
As there hasn’t been any recent activity, we plan to close the case soon unless we hear otherwise.
Please let us know at your earliest convenience. Thank you again for your valuable feedback. We truly appreciate your contribution in helping us improve.
Kind regards,
Alice
